src/OrbitClient.php

```diff
@@ -179,12 +182,16 @@ private function getUrl()
      */
     private function getRequestHeaders(array $options)
     {
-        return [
+        $headers = [
             'Accept' => 'application/ld+json',
             'Accept-Encoding' => 'gzip',
             'Accept-Language' => isset($options['language']) ? $options['language'] : 'en',
             'X-Orb-Variant' => isset($options['variant']) ? $options['variant'] : 'default',
         ];
+        if ($this->options['useCloudIdcta'] == true) {
```

Comment:
Nitpicky point of style, but could you make this just`if($this->options['useCloudIdcta'])`